modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -299,6 +278,16 @@ protected void containerIsStarted(InspectContainerResponse containerInfo) {
      * @return a new Remote Web Driver instance
      */
     public RemoteWebDriver getWebDriver() {
+        if (driver == null) {
+            if (capabilities == null) {
+                logger().warn("No capabilities provided - this will cause an exception in future versions. Falling back to ChromeOptions");
+                capabilities = new ChromeOptions();
+            }
+
+            driver = Unreliables.retryUntilSuccess(30, TimeUnit.SECONDS,
+                () -> Timeouts.getWithTimeout(10, TimeUnit.SECONDS,
+                    () -> new RemoteWebDriver(getSeleniumAddress(), capabilities)));
```

Comment:
nit: while we are on it, for the sake of readability, could you please use long lambdas here? :)

Something like:
```java
driver = Unreliables.retryUntilSuccess(30, TimeUnit.SECONDS, () -> {
    return Timeouts.getWithTimeout(10, TimeUnit.SECONDS, () -> {
        return new RemoteWebDriver(getSeleniumAddress(), capabilities);
    });
});

Reply:
done :)